.github/CODEOWNERS

```
@@ -0,0 +1,13 @@
+# The following volunteers have self-identified as subject matter experts
+# or interested parties over a particular area of the php-src source code.
+# While requesting a review from someone does not obligate that person to
+# review a pull request, these reviewers might have valuable knowledge of
+# the problem area and could aid in deciding whether a pull request is ready
+# for merging.
+#
+# For more information, see the GitHub CODEOWNERS documentation:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/ext/date/                  @derickr
+/ext/oci8/                  @cjbj
+/ext/pdo_oci/               @cjbj
```

I don't think we should list people without their explicit consent (@derickr already said he likes to  be informed about ext/date issues), except maybe for those five maintainers in EXTENSIONS which are listed for 2022.

```suggestion
/ext/com_dotnet/            @cmb69
/ext/gd/                    @cmb69
```

---

You can list me if it makes sense.  My affiliation is not a secret !

---

Updated the suggestion above.